src/main/java/nl/novi/TechItEasy/services/TelevisionService.java

```diff
@@ -0,0 +1,98 @@
+package nl.novi.TechItEasy.services;
+
+import nl.novi.TechItEasy.dtos.TelevisionOutputDto;
+import nl.novi.TechItEasy.dtos.TelevisionInputDto;
+import nl.novi.TechItEasy.models.Television;
+import nl.novi.TechItEasy.repositories.TelevisionRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class TelevisionService {
+
+
+    private final TelevisionRepository televisionRepository;
+    public TelevisionService(TelevisionRepository televisionRepository) {
+        this.televisionRepository=televisionRepository;
+    }
+
+    public List<TelevisionOutputDto> getAllTelevisions() {
+        return televisionRepository.findAll()
+                .stream()
+                .map(TelevisionService::toTelevisionDto)
+                .collect(Collectors.toList());
+    }
+
+    public TelevisionOutputDto getTelevisionById(int id) {
+        Television t = televisionRepository.findById(id).get();
+        return toTelevisionDto(t);
+    }
+
+    public TelevisionOutputDto saveTelevision(TelevisionInputDto television) {
+        Television t = toTelevision(television);
+        televisionRepository.save(t);
+        return toTelevisionDto(t);
+    }
+
+    public void deleteTelevision(int id) {
+        televisionRepository.deleteById(id);
+    }
+
+    public TelevisionOutputDto updateTelevision(int id, TelevisionInputDto newTv) {
+        Television t = toTelevision(newTv);
+        t.setId(id);
+        televisionRepository.save(t);
+
+
+        return toTelevisionDto(t);
```

Comment:
Hier heb je het id van t wel goed gezet, maar alsnog is het beter om de return waarde van de save te returnen. Stel dat er toch iets anders gaat dan verwacht, dan zie je dat niet.